app/controllers/application_controller.rb

```diff
@@ -1,2 +1,14 @@
 class ApplicationController < ActionController::Base
```

Comment:
- i can't create a post. Kindly work on this issue.
![Screen Shot 2021-08-12 at 05 34 45](https://user-images.githubusercontent.com/22258274/129182887-76dace05-c6a7-44d8-b4d9-401fbefa948d.png)